scripts/copyFiles/index.js

```diff
@@ -5,3 +5,4 @@ if (!shell.test('-d', './dist/assets')) {
 }
 shell.cp('-u', './server/assets/*.py', './dist/assets/');
 shell.cp('-u', './server/assets/pdf.worker.js', './node_modules/pdfjs-dist/build/');
+shell.cp('-u', './server/defaultConfig.json', './dist/bin');
```

Comment:
this will fail if path `/dist/bin` is not previously created.
Also, here `bin` is treated as a file instead of a directory